packages/webdriverio/src/commands/browser/custom$$.js

```diff
@@ -0,0 +1,43 @@
+/**
+ *
+ * The `customs$` allows you to use a custom strategy declared by using `browser.addLocatorStrategy`
+ *
+ * <example>
+    :pause.js
+    it('should get all the plugin wrapper buttons', () => {
+        browser.url('https://webdriver.io');
+        browser.addLocatorStrategy('myStrat', (selector) => {
+            return document.querySelectorAll(selector)
+        })
+
+        const pluginWrapper = browser.custom$$('myStrat', '.pluginWrapper')
+
+        console.log(pluginWrapper.length) // 4
+    });
+ * </example>
+ *
+ * @alias custom$$
+ * @param {String} strategyName
+ * @param {Any} strategyArguments
+ * @return {Element}
+ */
+import { runFnInFiberContext } from '@wdio/utils/build/shim'
+import { getElements } from '../../utils/getElementObject'
+
+async function custom$$ (strategyName, strategyArgument) {
+    const strategy = this.strategies.get(strategyName)
+
+    if (!strategy) {
+        throw Error('No strategy found for ' + strategyName)
+    }
+
+    let res = await this.execute(strategy, strategyArgument)
+
+    if (!Array.isArray(res)) {
+        res = [res]
+    }
+
+    return await getElements.call(this, strategy, res)
+}
+
+export default runFnInFiberContext(custom$$)
```

Comment:
This is not necessary. We don't do that with any command.

Reply:
@christian-bromann I wasn't able to run this in sync mode without the fiber context wrapper. Will look for another way.